cli_tools/gce_ovf_import/ovf_importer/ovf_importer_test.go

```diff
@@ -288,11 +260,22 @@ func Test_InstanceImport_SetupWorkflow_HappyCase_PreGA(t *testing.T) {
 	assert.Equal(t, params.Oauth, w.OAuthPath)
 	assert.Equal(t, params.Ce, w.ComputeEndpoint)
 	assert.Equal(t, params.ScratchBucketGcsPath, w.GCSPath)
-	assert.Len(t, w.Steps, 2, "Expect two steps: create-instance and cleanup")
+	if createMachineImage {
+		// Creating the machine image adds two steps:
+		//  1. Stop the instance
+		//  2. Create the GMI
```

Comment:
It also deletes the instance.

Reply:
Good point! Updated the comment, and added an assertion that the cleanup step includes the instance when a GMI is being created.